src/test/resources/com/puppycrawl/tools/checkstyle/checks/regexp/regexp/InputRegexpSemantic2.java

```diff
@@ -20,7 +20,7 @@
  * Test case for detecting simple semantic violations.
  * @author Lars Kühne
  **/
-class InputRegexpSemantic2 // violation
+class InputRegexpSemantic2
```

Comment:
This change looks weird but it is correct update, there is no violation here, test class also confirmed this

Reply:
same has been found for an input class using EmptyForInitializerPadCheckTest as well - https://github.com/checkstyle/checkstyle/pull/13966

Reply:
@romani how can this be? Do we have some bug in the violation comment parser?

Reply:
In other PR, input was used as part of an exception, so I assume that is why.

This PR does not look to be using in violation comment parser as it just uses `verify`.
https://github.com/checkstyle/checkstyle/blob/7c0950ee4aba6c6bd3ec56c617b879341a6a79ad/src/test/java/com/puppycrawl/tools/checkstyle/checks/regexp/RegexpCheckTest.java#L73

A lot of this test class uses `verifyWithInlineConfigParser`, so I am not sure why this one test isn't.

Is this part of an existing issue?

Reply:
Not listed here https://github.com/checkstyle/checkstyle/issues/11446

Reply:
I added it to this issue, better to fix this in separate PR

Reply:
It is listed in forbidden API suppression, so that is why it isn't listed in #11446 and why it is not done.
https://github.com/checkstyle/checkstyle/blob/c6a59c8a8c0f6c74c65df8f11bfd9a80c21c2cb2/pom.xml#L1795-L1797

Reply:
@rnveach thanks for investigating, @romani thanks for adding this to https://github.com/checkstyle/checkstyle/issues/11446.